src/App.test.tsx

```diff
@@ -2,10 +2,22 @@ import React from 'react';
 import {render} from '@testing-library/react';
 import App from './App';
 
+jest.mock('react-i18next', () => ({
+  // This mock makes sure any components using the translate hook
+  // can use it without a warning being shown
+  useTranslation: () => {
+    return {
+      t: (s: string) => s,
+      i18n: {
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
I think you can just say `no-empty-function` rather than `@typescript-eslint/no-empty-function` (just to be consistent with the other comments like this in the codebase

Reply:
Is required. It failed without